# Generated by Django 2.2.6 on 2021-08-16 00:59

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("reviews", "0006_auto_20210816_0012"),
    ]

    operations = [
        migrations.AlterField(
            model_name="user",
            name="username",
            field=models.CharField(
                error_messages={
                    "unique": "Пользователь с таким именем уже существует."
                },
                help_text="Необходимые. 150 символов или меньше.",
                max_length=150,
                unique=True,
                validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
                verbose_name="имя пользователя",
            ),
        ),
    ]
